### PR TITLE
Added content to the README, with anchor links and controls section

### PR DESCRIPTION
**Pull request by natemtate**
Hey added some stuff to the README

### CODE REVIEW

**Comment by rngoode1** [comment]
Refs #1

File: README.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+[2. Controls](#controls)
+[3. Build Info](#build-info)
+[4. Authors](#license)
+[5. License](#license)
+[6. Acknowledgements](#acknowledgements)
 
 ## Description
 
-Arachnivania is a 2D Metroidvania. In it you play as a spider trying to protect its burrow from invading Wasps. In the process you will discover new ways to fight and avoid enemies while trying to safe guard your burrow.
+Arachnivania is a 2D Metroidvania. You play as Widower, a warrior spider trying to protect his burrow from the invading Wasps. In the process you will discover new ways to fight and avoid enemies while protecting your family and burrow from harm.
```
Comment (rngoode1) [inline README.md:14]: Widower, badass name. I like it.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
 
 ## Controls
 
+* WASD - used for basic movement, A & D move left and right, respectively, while W & S move the camera slightly up or down.
+* SPACE - used to jump over obstacles and enemies
```
Comment (rngoode1) [inline README.md]: The prototype currently uses W as jump, just fyi

Comment (natemtate) [reply README.md]: oh okay thats fine, didn't know.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
 
-This project is a Godot-powered game using the native GDscript language. Art is created using Aseprite.
+This project is a Godot-powered game using the native GDscript language. Art is created using Aseprite, TileSetter, and GIMP.
+
+[1. Description](#desription)
+[2. Controls](#controls)
+[3. Build Info](#build-info)
+[4. Authors](#license)
+[5. License](#license)
+[6. Acknowledgements](#acknowledgements)
```
Comment (rngoode1) [inline README.md:11]: Do we need Authors, License and Ack. Is it possible to link to the actual file?

File: README.md (hunk 2 above)
Comment (natemtate) [reply README.md:19]: Oh yeah, you can link to whatever files you want with Markdown. What those links do at the top is scroll to the section of the page that is linked. So if you click on Acknowledgements at the top, its scrolls to the Acknowledgements part of the README

File: README.md (hunk 3 above)
Comment (natemtate) [reply README.md:11]: Oh yeah, you can link to whatever files you want with Markdown. What those links do at the top is scroll to the section of the page that is linked. So if you click on Acknowledgements at the top, its scrolls to the Acknowledgements part of the README

Comment (natemtate) [reply README.md:11]: Oops wrong comment spot

Comment (rngoode1) [reply README.md:11]: Yeah link them to the actual files instead of putting that info in the ReadMe

File: README.md (hunk 2 above)
Comment (rngoode1) [reply README.md]: No its fine, it doesnt really matter so long as it represents the final game

**Review by RyanAScott — CHANGES_REQUESTED** [review]
This creates a merge conflict

**Comment by rngoode1** [comment]
Rejecting per your request

**Pull request closed without merging**